src/cmd/build.js

```diff
@@ -40,11 +42,43 @@ export type PackageCreatorParams = {
 export type PackageCreatorFn =
     (params: PackageCreatorParams) => Promise<ExtensionBuildResult>;
 
+export type LocalizedNameParams = {
+  manifestData: ExtensionManifest,
+  messageData: any,
+};
+
+export function getDefaultLocalizedName(
+  {manifestData, messageData}: LocalizedNameParams
+  ): string {
+  let extensionName: string;
+
+  if (messageData) {
```

Comment:
This variable is not optional (because of how `LocalizedNameParams` typed it) so you don't need to check for its presence in an if statement. It is a required variable.